indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -305,19 +307,6 @@ def _process(self):
         return '', 204
 
 
-class RHResetEditable(RHContributionEditableRevisionBase):
```

Comment:
In case you want to keep this available to be called by the editing service (I don't know if there's any potential usecase for doing so at arbitrary times), you could check for `self.is_service_call` and abort if it's not a service call.

But I'm also fine with removing this altogether.

Reply:
yeah i'd rather remove it completely. this back and forth between indico and the service doesn't make much sense to me - it only seems to cause problems. we can simply add a "reset" flag to the service response in the other endpoints if needed

(just noticed a place where i missed btw, fixing it now)

Reply:
wondering if we should also remove the ability for the service to undo revisions...